src/main/java/software/amazon/awssdk/crt/io/DirectoryTraversal.java

```diff
@@ -0,0 +1,32 @@
+package software.amazon.awssdk.crt.io;
+
+/**
+ * This class wraps the directory traversal implementation provided by the CRT.
+ */
+public final class DirectoryTraversal {
```

Comment:
can we mark this "experimental?" or "internal use only"?

Reply:
Added the @deprecated annotation and corresponding documentation.